arm/Microsoft.KeyVault/vaults/keys/readme.md

```diff
@@ -0,0 +1,54 @@
+# Key Vault Key `[Microsoft.KeyVault/vaults/keys]`
+
+This module deploys a key vault key.
+
+## Resource Types
+
+| Resource Type | Api Version |
+| :-- | :-- |
+| `Microsoft.KeyVault/vaults/keys` | 2019-09-01 |
+
+## Parameters
+
+| Parameter Name | Type | Default Value | Possible Values | Description |
+| :-- | :-- | :-- | :-- | :-- |
+| `attributesEnabled` | bool | `True` |  | Optional. Determines whether the object is enabled. |
+| `attributesExp` | int | `-1` |  | Optional. Expiry date in seconds since 1970-01-01T00:00:00Z. |
+| `attributesNbf` | int | `-1` |  | Optional. Not before date in seconds since 1970-01-01T00:00:00Z. |
+| `cuaId` | string |  |  | Optional. Customer Usage Attribution id (GUID). This GUID must be previously registered |
+| `curveName` | string | `P-256` | `[P-256, P-256K, P-384, P-521]` | The elliptic curve name. |
```

Comment:
Add "Optional" in description. Was this not caught by the module test?

Reply:
Same for the other parameters under.

Reply:
It's not testing that, no. Probably worth adding. In this case I just forgot

Reply:
Fixed